apps/web/src/services/project/components/ProjectMemberTab.vue

```diff
@@ -171,7 +184,7 @@ watch(() => store.state.reference.project.items, (projects) => {
                          selectable
                          sortable
                          :fields="state.fields"
-                         :items="state.items"
+                         :items="state.refinedUserItems.slice(state.pageStart - 1, state.pageStart + state.pageLimit - 1)"
```

Comment:
nitpick: I think using computed state is more efficient, not to calculate when every bound states change.